cds/modules/webhooks/receivers.py

```diff
@@ -465,21 +479,21 @@ def delete(self, event):
             object_version = as_object_version(event.payload['version_id'])
             ObjectVersionTag.query.filter(
                 ObjectVersionTag.object_version == object_version,
-                (ObjectVersionTag.key == '_event_id') |
-                (ObjectVersionTag.key == 'preview')
-            ).delete()
+                ObjectVersionTag.key.in_(['_event_id', 'preview',
+                                          'media_type', 'context_type'])
+            ).delete(synchronize_session=False)
```

Comment:
I was getting this error `InvalidRequestError: Could not evaluate current criteria in Python.  Specify 'fetch' or False for the synchronize_session parameter.`